hal/src/thumbv6m/clock.rs

```diff
@@ -180,23 +180,35 @@ impl GenericClockController {
         }
 
         // Feed 32khz into the DFLL48
-        state.enable_clock_generator(DFLL48, GCLK1);
+        #[cfg(not(feature = "samd20"))]
+        state.enable_clock_generator(ClockId::DFLL48, GCLK1);
```

Comment:
why `DFLL48` here but `DFLL48M in `state.set_gclk_divider_and_source`?

Reply:
The svd file defines it that way. Maybe I can try adding a replacement rule in the xslt, I'll check that

Reply:
I just made a replacement in the xslt so the naming is now identical :)